Index: app/alertmaker/.gitignore
===================================================================
diff --git a/app/alertmaker/.gitignore b/app/alertmaker/.gitignore
deleted file mode 100644
--- a/app/alertmaker/.gitignore	(revision 89af651a3be100391e0d1be69e1db646fd397298)
+++ /dev/null	(revision 89af651a3be100391e0d1be69e1db646fd397298)
@@ -1,1 +0,0 @@
-/build
\ No newline at end of file
Index: app/alertmaker/src/test/java/com/iorgana/alertmaker/ExampleUnitTest.java
===================================================================
diff --git a/app/alertmaker/src/test/java/com/iorgana/alertmaker/ExampleUnitTest.java b/app/alertmaker/src/test/java/com/iorgana/alertmaker/ExampleUnitTest.java
deleted file mode 100644
--- a/app/alertmaker/src/test/java/com/iorgana/alertmaker/ExampleUnitTest.java	(revision 89af651a3be100391e0d1be69e1db646fd397298)
+++ /dev/null	(revision 89af651a3be100391e0d1be69e1db646fd397298)
@@ -1,17 +0,0 @@
-package com.iorgana.alertmaker;
-
-import org.junit.Test;
-
-import static org.junit.Assert.*;
-
-/**
- * Example local unit test, which will execute on the development machine (host).
- *
- * @see <a href="http://d.android.com/tools/testing">Testing documentation</a>
- */
-public class ExampleUnitTest {
-    @Test
-    public void addition_isCorrect() {
-        assertEquals(4, 2 + 2);
-    }
-}
\ No newline at end of file
Index: app/alertmaker/src/androidTest/java/com/iorgana/alertmaker/ExampleInstrumentedTest.java
===================================================================
diff --git a/app/alertmaker/src/androidTest/java/com/iorgana/alertmaker/ExampleInstrumentedTest.java b/app/alertmaker/src/androidTest/java/com/iorgana/alertmaker/ExampleInstrumentedTest.java
deleted file mode 100644
--- a/app/alertmaker/src/androidTest/java/com/iorgana/alertmaker/ExampleInstrumentedTest.java	(revision 89af651a3be100391e0d1be69e1db646fd397298)
+++ /dev/null	(revision 89af651a3be100391e0d1be69e1db646fd397298)
@@ -1,26 +0,0 @@
-package com.iorgana.alertmaker;
-
-import android.content.Context;
-
-import androidx.test.platform.app.InstrumentationRegistry;
-import androidx.test.ext.junit.runners.AndroidJUnit4;
-
-import org.junit.Test;
-import org.junit.runner.RunWith;
-
-import static org.junit.Assert.*;
-
-/**
- * Instrumented test, which will execute on an Android device.
- *
- * @see <a href="http://d.android.com/tools/testing">Testing documentation</a>
- */
-@RunWith(AndroidJUnit4.class)
-public class ExampleInstrumentedTest {
-    @Test
-    public void useAppContext() {
-        // Context of the app under test.
-        Context appContext = InstrumentationRegistry.getInstrumentation().getTargetContext();
-        assertEquals("com.iorgana.alertmaker.test", appContext.getPackageName());
-    }
-}
\ No newline at end of file
Index: app/alertmaker/proguard-rules.pro
===================================================================
diff --git a/app/alertmaker/proguard-rules.pro b/app/alertmaker/proguard-rules.pro
deleted file mode 100644
--- a/app/alertmaker/proguard-rules.pro	(revision 89af651a3be100391e0d1be69e1db646fd397298)
+++ /dev/null	(revision 89af651a3be100391e0d1be69e1db646fd397298)
@@ -1,21 +0,0 @@
-# Add project specific ProGuard rules here.
-# You can control the set of applied configuration files using the
-# proguardFiles setting in build.gradle.
-#
-# For more details, see
-#   http://developer.android.com/guide/developing/tools/proguard.html
-
-# If your project uses WebView with JS, uncomment the following
-# and specify the fully qualified class name to the JavaScript interface
-# class:
-#-keepclassmembers class fqcn.of.javascript.interface.for.webview {
-#   public *;
-#}
-
-# Uncomment this to preserve the line number information for
-# debugging stack traces.
-#-keepattributes SourceFile,LineNumberTable
-
-# If you keep the line number information, uncomment this to
-# hide the original source file name.
-#-renamesourcefileattribute SourceFile
\ No newline at end of file
Index: app/alertmaker/build.gradle
===================================================================
diff --git a/app/alertmaker/build.gradle b/app/alertmaker/build.gradle
deleted file mode 100644
--- a/app/alertmaker/build.gradle	(revision 89af651a3be100391e0d1be69e1db646fd397298)
+++ /dev/null	(revision 89af651a3be100391e0d1be69e1db646fd397298)
@@ -1,35 +0,0 @@
-plugins {
-    id 'com.android.library'
-}
-
-android {
-    namespace 'com.iorgana.alertmaker'
-    compileSdk 33
-
-    defaultConfig {
-        minSdk 21
-
-        testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
-        consumerProguardFiles "consumer-rules.pro"
-    }
-
-    buildTypes {
-        release {
-            minifyEnabled false
-            proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
-        }
-    }
-    compileOptions {
-        sourceCompatibility JavaVersion.VERSION_1_8
-        targetCompatibility JavaVersion.VERSION_1_8
-    }
-}
-
-dependencies {
-
-    implementation 'androidx.appcompat:appcompat:1.6.1'
-    implementation 'com.google.android.material:material:1.11.0'
-    testImplementation 'junit:junit:4.13.2'
-    androidTestImplementation 'androidx.test.ext:junit:1.1.5'
-    androidTestImplementation 'androidx.test.espresso:espresso-core:3.5.1'
-}
\ No newline at end of file
Index: app/alertmaker/src/main/AndroidManifest.xml
===================================================================
diff --git a/app/alertmaker/src/main/AndroidManifest.xml b/app/alertmaker/src/main/AndroidManifest.xml
deleted file mode 100644
--- a/app/alertmaker/src/main/AndroidManifest.xml	(revision 89af651a3be100391e0d1be69e1db646fd397298)
+++ /dev/null	(revision 89af651a3be100391e0d1be69e1db646fd397298)
@@ -1,4 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
-<manifest xmlns:android="http://schemas.android.com/apk/res/android">
-
-</manifest>
\ No newline at end of file
Index: settings.gradle
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>pluginManagement {\r\n    repositories {\r\n        google()\r\n        mavenCentral()\r\n        gradlePluginPortal()\r\n    }\r\n}\r\ndependencyResolutionManagement {\r\n    repositoriesMode.set(RepositoriesMode.FAIL_ON_PROJECT_REPOS)\r\n    repositories {\r\n        google()\r\n        mavenCentral()\r\n        maven { url \"https://jitpack.io\" }\r\n        maven { url \"https://maven2.google.com\"}\r\n        jcenter()\r\n    }\r\n}\r\n\r\nrootProject.name = \"Iorgana Libraries\"\r\ninclude ':app'\r\ninclude ':AlertMaker'\r\ninclude ':DroidHelpers'\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/settings.gradle b/settings.gradle
--- a/settings.gradle	(revision 89af651a3be100391e0d1be69e1db646fd397298)
+++ b/settings.gradle	(date 1705067687881)
@@ -10,6 +10,7 @@
     repositories {
         google()
         mavenCentral()
+        mavenLocal()
         maven { url "https://jitpack.io" }
         maven { url "https://maven2.google.com"}
         jcenter()
Index: AlertMaker/.idea/gradle.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><?xml version=\"1.0\" encoding=\"UTF-8\"?>\r\n<project version=\"4\">\r\n  <component name=\"GradleSettings\">\r\n    <option name=\"linkedExternalProjectsSettings\">\r\n      <GradleProjectSettings>\r\n        <option name=\"testRunner\" value=\"GRADLE\" />\r\n        <option name=\"distributionType\" value=\"DEFAULT_WRAPPED\" />\r\n        <option name=\"externalProjectPath\" value=\"$PROJECT_DIR$\" />\r\n        <option name=\"gradleJvm\" value=\"jbr-17\" />\r\n      </GradleProjectSettings>\r\n    </option>\r\n  </component>\r\n</project>
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/AlertMaker/.idea/gradle.xml b/AlertMaker/.idea/gradle.xml
--- a/AlertMaker/.idea/gradle.xml	(revision 89af651a3be100391e0d1be69e1db646fd397298)
+++ b/AlertMaker/.idea/gradle.xml	(date 1705072100426)
@@ -1,5 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
+  <component name="GradleMigrationSettings" migrationVersion="1" />
   <component name="GradleSettings">
     <option name="linkedExternalProjectsSettings">
       <GradleProjectSettings>
Index: AlertMaker/.idea/misc.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><?xml version=\"1.0\" encoding=\"UTF-8\"?>\r\n<project version=\"4\">\r\n  <component name=\"ExternalStorageConfigurationManager\" enabled=\"true\" />\r\n  <component name=\"ProjectRootManager\" version=\"2\" project-jdk-name=\"jbr-17\" project-jdk-type=\"JavaSDK\">\r\n    <output url=\"file://$PROJECT_DIR$/build/classes\" />\r\n  </component>\r\n  <component name=\"ProjectType\">\r\n    <option name=\"id\" value=\"Android\" />\r\n  </component>\r\n</project>
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/AlertMaker/.idea/misc.xml b/AlertMaker/.idea/misc.xml
--- a/AlertMaker/.idea/misc.xml	(revision 89af651a3be100391e0d1be69e1db646fd397298)
+++ b/AlertMaker/.idea/misc.xml	(date 1705072073572)
@@ -1,4 +1,3 @@
-<?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
   <component name="ExternalStorageConfigurationManager" enabled="true" />
   <component name="ProjectRootManager" version="2" project-jdk-name="jbr-17" project-jdk-type="JavaSDK">
Index: AlertMaker/.gitignore
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>/build
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/AlertMaker/.gitignore b/AlertMaker/.gitignore
--- a/AlertMaker/.gitignore	(revision 89af651a3be100391e0d1be69e1db646fd397298)
+++ b/AlertMaker/.gitignore	(date 1705076154558)
@@ -1,1 +1,0 @@
-/build
\ No newline at end of file
Index: DroidHelpers/.gitignore
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>/build
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/DroidHelpers/.gitignore b/DroidHelpers/.gitignore
--- a/DroidHelpers/.gitignore	(revision 89af651a3be100391e0d1be69e1db646fd397298)
+++ b/DroidHelpers/.gitignore	(date 1705076154558)
@@ -1,1 +1,0 @@
-/build
\ No newline at end of file
diff --git a/app/alertmaker/consumer-rules.pro b/app/alertmaker/consumer-rules.pro
deleted file mode 100644
